Add embed block

Toto-Tuo opened the pull request:
Source: https://github.com/hlxsites/eder-group/tree/3ea2c4c1fdccc1dc05fb71b903fada0563fc7929/blocks/embed
	new file:   blocks/embed/embed-facebook.js
	new file:   blocks/embed/embed-instagram.js
	new file:   blocks/embed/embed-youtube-channel.js
	new file:   blocks/embed/embed.css
	new file:   blocks/embed/embed.js

Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #6-add-embed-block

Test URLs:
- Before: https://main--fresenius--hlxsites.hlx.page/
- After: https://6-issue-block-embed--fresenius--hlxsites.hlx.page/

Toto-Tuo committed:
Added embed block
  Source: https://github.com/hlxsites/eder-group/tree/3ea2c4c1fdccc1dc05fb71b903fada0563fc7929/blocks/embed
	new file:   blocks/embed/embed-facebook.js
	new file:   blocks/embed/embed-instagram.js
	new file:   blocks/embed/embed-youtube-channel.js
	new file:   blocks/embed/embed.css
	new file:   blocks/embed/embed.js

Toto-Tuo commented:
Taking a look at the failed build.

helms-charity commented on blocks/embed/embed.js:
```diff
@@ -0,0 +1,140 @@
+import { toClassName } from '../../scripts/lib-franklin.js';
+import { customDecorateIcons } from '../../scripts/scripts.js';
```
I think you can change this to 
import { decorateIcons } from '../../scripts/lib-franklin.js';

And then find and replace all instances of "customDecorateIcons" to "decorateIcons" in these new files. We'll have to look deeper and test things but we don't need to support the 'custom' icon libraries that the other site is using.

helms-charity commented on blocks/embed/embed-facebook.js:
```diff
@@ -0,0 +1,9 @@
+import { loadScript } from '../../scripts/scripts.js';
```
You will need to add the "loadScript" function as seen at https://github.com/hlxsites/eder-group/blob/main/scripts/scripts.js#L19 into this repo's scripts.js. All it does is put the correlating 3rd party embed script into the Head if these services are used on the page.

Toto-Tuo committed:
fix: remove custom decorateIcons , add loadScript in scripts.js
	modified:   blocks/embed/embed.js
	modified:   package-lock.json
	modified:   scripts/scripts.js

Toto-Tuo committed:
Merge branch 'main' of https://github.com/hlxsites/fresenius into 6-issue-block-embed
